README.md

```diff
@@ -1,4 +1,4 @@
-# HTML template for the AsyncAPI Generator
+# HTML template for the AsyncAPI Generator-An Amazing Project
```

Comment:
Thanks a lot for the good words ❤️ but we would prefer to avoid such statements in headings in the readme. Please revert this change. The other change in `Supported Parameters` is of course appreciated